.github/workflows/bump-channel-version/action.yml

```diff
@@ -8,6 +8,10 @@ inputs:
   CLEAN_PACKAGE_VERSION:
     required: true
     description: 'The current package version, e.g. "3.11.7".'
+  OVERRIDE_PACKAGE_VERSION:
+    required: false
+    default: "false"
```

Comment:
The default value was added as the `OVERRIDE_PACKAGE_VERSION` variable must contain a value. 
 Or else the 
 

```
if [[ ${{ inputs.OVERRIDE_PACKAGE_VERSION }} == "true" ]]; then
              echo "PACKAGE_VERSION=${ELEMENTOR_CHANNEL_PACKAGE_VERSION}" >> $GITHUB_ENV
            fi

```
Will never pass.